Essentials/src/main/resources/config.yml

```
@@ -341,24 +341,30 @@ pastebin-createkit: false
 # Color is not an actual sign, it's for enabling using color codes on signs, when the correct permissions are given.
 
 enabledSigns:
-  #- color
+  #- anvil
   #- balance
   #- buy
+  #- cartography
+  #- color
   #- sell
   #- trade
   #- free
   #- disposal
   #- warp
   #- kit
+  #- loom
   #- mail
   #- enchant
   #- gamemode
+  #- grindstone
   #- heal
   #- info
   #- spawnmob
+  #- smithing
   #- repair
   #- time
   #- weather
+  #- workbench
```

```suggestion
  #- color
  #- balance
  #- buy
  #- sell
  #- trade
  #- free
  #- warp
  #- kit
  #- mail
  #- enchant
  #- gamemode
  #- heal
  #- info
  #- spawnmob
  #- repair
  #- time
  #- weather
  #- anvil
  #- cartography
  #- disposal
  #- grindstone
  #- loom
  #- smithing
  #- workbench
```
This moves all the inventory signs to the end, rather than mixing them in with the existing groupings